### ISSUE TITLE
Make default files available somewhere we own. 

### ISSUE THREAD

**Issue by vigliensoni** [lingxiaoyang/Tension-Analysis#6]
Make sure the files

- model.h5
- stanford-corenlp-full-2018-02-27.zip

are available in a place that we can always access. If the size is not big, perhaps another repo that we can use as a submodule?

**Comment by lingxiaoyang** [comment]
stanford-corenlp-full-2018-02-27.zip  I think this file can be downloaded and assembled from Stanford CoreNLP official page. 
Maybe putting model.h5 right into this repo itself? I think 64MB is acceptable.

**Comment by vigliensoni** [comment]
Absolutely we can put it, just in case the Stanford page is not available
anymore. Perhaps as a submodule if we want to tidy up things? (Not
mandatory, though)

**Comment by lingxiaoyang** [comment]
I think the Stanford one is pretty stable. Its repo seems popular on GitHub: https://github.com/stanfordnlp/CoreNLP So I don't worry about it.
I'll figure out how `stanford-corenlp-full-2018-02-27.zip` is packaged and put that into README.

**Comment by lingxiaoyang** [comment]
OK, the stanford zip is just an unedited copy of official release of Stanford CoreNLP 3.9.1:
![image](https://user-images.githubusercontent.com/8630726/60845996-07dece80-a1ac-11e9-94bc-9f7e5367532f.png)
I'll initialize this with Dockerfile.

**Comment by lingxiaoyang** [comment]
OK, I included `model.h5` into the repo. The message I received during commit was:

````
remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
remote: warning: See http://git.io/iEPt8g for more information.
remote: warning: File data/models/model.h5 is 63.53 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
````

Fortunately the file doesn't exceed GH limit of 100MB, so still acceptable. 

I failed to use the Git Large File Storage because this repo is a public fork. The error was: batch response: @lingxiaoyang can not upload new objects to public fork mat3rial/Tension-Analysis

Thus at this point, I think it's the simplest to put this single file into the repo itself. If we happen to update `model.h5` frequently in the future, we'll amend the commit 3703a4e7b7424a71aaf04096574c8b80fac1dc7a to remove the large file and rebase everything after it.
